infra/event-sync/src/main/java/com/simprints/infra/eventsync/event/remote/models/session/ApiSessionScope.kt

```diff
@@ -0,0 +1,48 @@
+package com.simprints.infra.eventsync.event.remote.models.session
+
+import androidx.annotation.Keep
+import com.simprints.infra.events.event.domain.models.Event
+import com.simprints.infra.events.event.domain.models.session.SessionScope
+import com.simprints.infra.eventsync.event.remote.models.ApiEvent
+import com.simprints.infra.eventsync.event.remote.models.ApiModality
+import com.simprints.infra.eventsync.event.remote.models.fromDomainToApi
+
+@Keep
+internal data class ApiSessionScope(
+    val id: String,
+    val projectId: String,
+    val startTime: Long,
+    val endTime: Long,
+    val endCause: ApiSessionEndCause,
+    val modalities: List<ApiModality>,
+    val sidVersion: String,
```

Comment:
I notice the "libSimprintsVersion" field that used to appear in SessionCapture_v1 event is missing.
Is this intentional?

Reply:
Missed it from the swagger, will fix it.